CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ Entries in this file should be limited to:
 Please avoid adding duplicate information across this changelog and JIRA/doc input pages.
 
 ## [NEXT RELEASE]
+- ROX-11348: The email notifier now allows for unauthenticated smtp.
+  The email notifier now allows for unauthenticated SMTP. By default, 
```

Comment:
"The email notifier now allows for unauthenticated SMTP" is repeated twice. Also put both sentences into one line?